### PR TITLE
DEV 2298 different prefix for different branch

### PR DESCRIPTION
**Pull request by qqiao2024**

### CODE REVIEW

**Review by stilesj-uchicago — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[metadata]
+name = gdcmodels
+description = "Stores and serves GDC data models defined in static YAML files.
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = NCI GDC
+author_email = gdc_dev_questions-aaaaae2lhsbell56tlvh3upgoq@cdis.slack.com
+license = Apache Software License 2.0
+license_file = LICENSE
+url = https://github.com/NCI-GDC/gdcdatamodel2
```
Comment (wrkhenddher) [inline setup.cfg]: gdcdatamodel **2**?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+keywords = python3 gdcdatamodels gdcdictionary plaster
+classifiers =
+    Intended Audience :: Developers
+    Programming Language :: Python
+    Programming Language :: Python :: 3.8
```
Comment (wrkhenddher) [inline setup.cfg:15]: 3.7 too?

File: tox.ini [hunk 3]
```diff
@@ -25,16 +25,18 @@ commands = python-codacy-coverage -r coverage.xml
 [testenv:publish]
 changedir =
 passenv =
-  TWINE_*
+    TWINE_*
+    CI_COMMIT_*
 skip_install=true
 deps =
-    setuptools_scm
+    setuptools_scm==8.0.4
```
Comment (wrkhenddher) [inline tox.ini:32]: do we really need to pin setuptools_scm?
If so, maybe we should pin it in pyproject.toml instead?

File: tox.ini [hunk 4]
```diff
@@ -25,16 +25,18 @@ commands = python-codacy-coverage -r coverage.xml
...
+    setuptools_git_versioning<2
```
Comment (wrkhenddher) [inline tox.ini:33]: maybe we should pin it in pyproject.toml instead?

**Review by wrkhenddher — APPROVED** [review]

File: setup.cfg (hunk 2 above)
Comment (qqiao2024) [reply setup.cfg:15]: 3.7 is removed from this repo

File: tox.ini (hunk 3 above)
Comment (qqiao2024) [reply tox.ini:32]: already in there, second line.

File: tox.ini (hunk 4 above)
Comment (qqiao2024) [reply tox.ini:33]: already in there, second line.

**Pull request merged**